helm/reana/templates/reana-db.yaml

```diff
@@ -73,7 +73,11 @@ spec:
           hostPath:
             path: {{ .Values.shared_storage.hostpath.root_path }}
           {{- end }}
-      {{- if .Values.node_label_infrastructure }}
+      {{- if .Values.node_label_infrastructuredb }}
+      {{- $full_label := split "=" .Values.node_label_infrastructuredb }}
+      nodeSelector:
+        {{ $full_label._0 }}: {{ $full_label._1 }}
```

Comment:
Is it possible to reuse these two lines to avoid repetition? I wonder if something like this would work:

```yaml
      {{- if .Values.node_label_infrastructuredb }}
      {{- $full_label := split "=" .Values.node_label_infrastructuredb }}
      {{- else if .Values.node_label_infrastructure }}
      {{- $full_label := split "=" .Values.node_label_infrastructure }}
      {{- end }}
      {{- if $full_label }}
      nodeSelector:
        {{ $full_label._0 }}: {{ $full_label._1 }}
      {{- end }}
```